frameworks/projects/Jewel/src/main/royale/org/apache/royale/jewel/beads/views/VirtualComboBoxPopUpView.as

```diff
@@ -58,6 +59,11 @@ package org.apache.royale.jewel.beads.views
                 _list = new VirtualList();
 				_list.addEventListener("beadsAdded", beadsAddedHandler);
             }
+
+			if((_strand as VirtualComboBoxPopUp).itemRenderer)
```

Comment:
I think this changes should be made in setter of strand if possible. I would also make it in following way:

```
var virtualComboBoxPopup:VirtualComboBoxPopUp = _strand as VirtualComboBoxPopUp;
if (virtualComboBoxPopup && virtualComboBoxPopup.itemRenderer)
{
     list.itemRenderer = virtualComboBoxPopup.itemRenderer;
}
```

Reply:
I looked at how it was fixed in the ComboBoxPopUpView and I thought I had "replicated" it but I didn't realise it was inside the "if(!_list)", I'll be more attentive next time (... the truth is that I was annoyed by the "list" but I didn't question it because I thought I had done it anyway :) )

I've included it in the "strand setter", as you indicated and verified the operation.

Hiedra.